web-frontend/src/pages/TransferCoins/SendCoins/Send.tsx

```diff
@@ -106,9 +106,10 @@ function Send(props: SendProps) {
   }
 
   function handleFees(num: string) {
+    // check if '.' is present
+    if (num.includes('.')) return;
```

Comment:
why do you return nothing if . is present? if it's input validation, you should show an error?

Reply:
we're blocking the input if a '.' is added
the error is still set in case